src/components/security_manager/src/crypto_manager_impl.cc

```diff
@@ -234,7 +234,7 @@ bool CryptoManagerImpl::OnCertificateUpdated(const std::string& data) {
 }
 
 SSLContext* CryptoManagerImpl::CreateSSLContext() {
-  if (NULL == context_ || IsCertificateUpdateRequired()) {
+  if (NULL == context_) {
```

Comment:
Currently the main problem is that the SDL should wait PTU finished before send any answer.